### PR TITLE
Refactor code to reference "verifiable credential" as VC

### LINKED ISSUE #75: Refactor code to reference "verifiable credential" as VC
In #18 @pmigueld mentioned:

> However we still refer to them as vid in the code so we should also update them eventually to be more clear right? Because we currently have a case of referencing the UIN of a VID (vid.uin) which no longer seems to make sense?

This outcome of this issue to perform the refactoring of this.

### PR DESCRIPTION
**Pull request by danicaerediano**
Closes #75

### CODE REVIEW

File: machines/settings.ts [hunk 1]
```diff
@@ -2,23 +2,23 @@ import { ContextFrom, EventFrom, send, sendUpdate, StateFrom } from 'xstate';
...
-import { VIDLabel } from '../types/vc';
+import { VCLabel } from '../types/vc';
 import { StoreEvents } from './store';
 
 const model = createModel(
   {
     serviceRefs: {} as AppServices,
     name: '',
-    vidLabel: {
+    VCLabel: {
```
Comment (pmigueld) [inline machines/settings.ts:12]: Can you change this (and other references) to `vcLabel` to distinguish from the `VCLabel` type?

File: screens/Home/HomeScreenController.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
 import { useInterpret, useSelector } from '@xstate/react';
 import { useContext, useEffect, useRef } from 'react';
-import { selectVidLabel } from '../../machines/settings';
+import { selectVCLabel } from '../../machines/settings';
```
Comment (pmigueld) [inline screens/Home/HomeScreenController.ts:3]: Let's keep it consistent with camelCase `selectVcLabel`.

File: screens/Home/MyVcs/DownloadingVcModalController.ts [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import { useSelector } from '@xstate/react';
 import { useContext } from 'react';
-import { selectVidLabel } from '../../../machines/settings';
+import { selectVCLabel } from '../../../machines/settings';
 import { GlobalContext } from '../../../shared/GlobalContext';
 
-export function useDownloadingvidModal() {
+export function useDownloadingvcModal() {
```
Comment (pmigueld) [inline screens/Home/MyVcs/DownloadingVcModalController.ts:6]: I think this was a typo from me, please fix to `useDownloadingVcModal`.

**Review by pmigueld — CHANGES_REQUESTED** [review]
@nicholemnl please check if I missed anything. Best to have more eyes on big refactors like this.

File: screens/Request/RequestScreenController.ts [hunk 4]
```diff
@@ -39,15 +39,15 @@ export function useRequestScreen({ navigation }: MainRouteProps) {
...
 
   let statusMessage = '';
   if (isWaitingForConnection) {
     statusMessage = 'Waiting for connection...';
   } else if (isExchangingDeviceInfo) {
     statusMessage = 'Exchanging device info...';
-  } else if (isWaitingForVid) {
-    statusMessage = `Connected to device. Waiting for ${vidLabel.singular}...`;
+  } else if (isWaitingForVc) {
+    statusMessage = `Connected to device. Waiting for ${VCLabel.singular}...`;
```
Comment (nicholemnl) [inline screens/Request/RequestScreenController.ts:50]: Shouldn't this be small vcLabel?

**Review by nicholemnl — APPROVED** [review]

**Review by pmigueld — APPROVED** [review]

**Pull request merged**